src/Catalog/Icons/IconProcessor.cs

```diff
@@ -0,0 +1,274 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using NuGet.Common;
+using NuGet.Services.Metadata.Catalog.Persistence;
+
+namespace NuGet.Services.Metadata.Catalog.Icons
+{
+    public class IconProcessor : IIconProcessor
+    {
+        private const string DefaultCacheControl = "max-age=120";
```

Comment:
>DefaultCacheControl [](start = 29, length = 19)

Seems like a low cache time but I guess we can override it in CDN (or do we already??(

Reply:
We use the same for nuspec and nupkg in flat container.